src/test/java/seedu/task/logic/parser/FindCommandParserTest.java

```diff
@@ -27,16 +34,29 @@ public void parse_validArgs_returnsFindCommand() {
         FindCommand expectedFindCommand =
                 new FindCommand(new NameContainsKeywordsPredicate("Alice Bob"));
         assertParseSuccess(parser, " n/Alice Bob", expectedFindCommand);
+        List<String> names = new ArrayList<>();
+        names.add("Bob");
+        names.add("Alice");
         FindCommand expectedFindCommand2 =
-                new FindCommand(new NameContainsKeywordsPredicate("Alice \n \t Bob"));
-        // multiple whitespaces between keywords
-        assertParseSuccess(parser, " \n n/Alice \n \t Bob  \t", expectedFindCommand2);
+            new FindCommand(new NameContainsAllKeywordsPredicate(names));
+        assertParseSuccess(parser, " all/hi n/Alice n/Bob", expectedFindCommand2);
```

Comment:
Something for consideration, all/ should not allow words after it as it should be just used for flagging.

Reply:
for now this is the behaviour but we will change in the future